cmd/cloud/installation.go

```diff
@@ -383,6 +384,32 @@ var installationListCmd = &cobra.Command{
 	},
 }
 
+var installationsGetStatuses = &cobra.Command{
+	Use:   "status",
+	Short: "Get status information for all installations.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		installationsStatus, err := client.GetInstallationsStatus()
+		if err != nil {
+			return errors.Wrap(err, "failed to query installation status")
+		}
+		if installationsStatus == nil {
+			return nil
+		}
+
+		err = printJSON(installationsStatus)
+		if err != nil {
```

Comment:
This would definitely work, but we have internally settled on a style where we don't use this line-saver. I will probably keep this as is unless we decide to go through the code and change all instances of this.

Reply:
:+1: I'm always suggesting this change but if you've set that style here is ok, I'll be awesome to have a style guide like the one the server team has :D

Reply:
Totally agree on this! Let's make sure that gets done. Thanks!